### DISCUSSION TITLE
Cannot perform lang ID

### DISCUSSION THREAD

**Discussion by Abdurrafey-Siddiqui** [openai/whisper#833] [Q&A]
When i do something like base.en or medium.en I get:

`This model doesn't have language tokens so it can't perform lang id`

**Comment by Coolshanlan** [comment]
I fixed the problem, you can directly specify the language ID (en) in the language parameter of `DecodingOptions`, there will be no error
Like this:
```
options = pywhisper.DecodingOptions(fp16 = False,language='en')
result = pywhisper.decode(model, mel, options)
```
